Project 2/four-card-feature/style.css

```diff
@@ -0,0 +1,152 @@
+body{
+    margin: 0;
+    padding: 0;
+    font-family: 'Poppins', sans-serif;
+    background-color: hsl(0, 0%, 98%);
+    font-size:15px; 
+    
+}
+.reliable{
+    display: flex;
+    height: auto;
+    flex-direction: column;
+    width: 30%;
+    text-align: center;
+    margin-top:40px; 
+    margin-bottom: 40px; 
+    margin-right:auto; 
+    margin-left: auto; 
+    flex-wrap:wrap;  
+    
+}
+.reliable>*{
+    margin: 0;
+} 
+
+.reliable>h1:first-of-type{
+    font-weight: 200;
+}
+
+.reliable>h1:nth-child(2){
+    color:hsl(234, 12%, 34%);
+}
+.reliable>p:first-of-type{
+    font-weight: 400;
+    color: hsl(229, 6%, 66%);
+}
+
+p {
+    color: hsl(229, 6%, 66%);  
+    font-weight:400; 
+    font-size: 12px; 
+}
+
+.supervisor{
+    width: 80%;
+    margin-right:auto; 
+    margin-left:auto; 
+    display: flex;
+    align-items: center;
+    justify-content: space-evenly;
+    flex-wrap: wrap;
+    gap: 25px; 
+    color:hsl(234, 12%, 34%);
+}
+.square{
+    width: 100%;
+    height: 250px; 
+    background-color: white;
+    border-top: 4px solid hsl(180, 62%, 55%);
+    border-radius: 8px;
+    padding: 15px 30px;
+    box-sizing: border-box;
+    box-shadow:  rgba(149, 157, 165, 0.2) 0px 8px 24px; 
+    gap: 10px; 
+    flex:1; 
+}
+
+.square img{
+   float: right; 
+    
+}
+
+.team>.square:first-child{
+    border-color: red;
+}
+.team>.square:nth-child(2n) {
+    border-color: orange;
+
+}
+.supervisor>.column:last-child .square{
+    border-color: blue;
+}
+
+.team{
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    align-items: center;
+    gap:20px;
+    flex:1;
+}
+
+.column {
+    flex: 1;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    align-items: center;
+    gap:20px;
+
```

Comment:
kod jest dokładnie taki sam, jak klasy .team powyżej :) może dałoby się to uwspólnić?